README.md

```diff
@@ -390,6 +390,11 @@ Variable | Description | Default value
 `ENABLE_HTTP_REDIRECT` | Redirects HTTP traffic to HTTPS | 1
 `LOG_LEVEL` | Controls which logs are output from prosody and associated modules | info
 
+#### Jitsi Meet `config.js`
```

Comment:
This needs to be moved to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md

Reply:
I will do this after it's clear how the final solution will look like.